Code/autopkglib/github/__init__.py

```diff
@@ -15,24 +16,31 @@
 # limitations under the License.
 """Routines for working with the GitHub API"""
 
-from __future__ import print_function
-
 import json
 import os
 import re
-import subprocess
+import tempfile
 
-from autopkglib import curl_cmd, get_pref, log, log_err
+from autopkglib import get_pref, log, log_err
+from autopkglib.URLGetter import URLGetter
 
 BASE_URL = "https://api.github.com"
 TOKEN_LOCATION = os.path.expanduser("~/.autopkg_gh_token")
 
 
-class GitHubSession(object):
+class GitHubSession(URLGetter):
     """Handles a session with the GitHub API"""
 
-    def __init__(self):
+    def __init__(self, curl_path=None, curl_opts=None):
```

Comment:
Added:
- Calling superclass constructor
- initializing varibles
- Supports `curl_path` and `curl_opts` parameters